3811c3811
< v3 = staticinvoke <org.apache.hadoop.hdfs.server.datanode.DataNode$lambda_checkDiskErrorAsync_0__45: org.apache.hadoop.hdfs.server.datanode.checker.DatasetVolumeChecker$Callback bootstrap$(org.apache.hadoop.hdfs.server.datanode.DataNode)>(v0);
---
> v3 = staticinvoke <org.apache.hadoop.hdfs.server.datanode.DataNode$lambda_checkDiskErrorAsync_0__85: org.apache.hadoop.hdfs.server.datanode.checker.DatasetVolumeChecker$Callback bootstrap$(org.apache.hadoop.hdfs.server.datanode.DataNode)>(v0);
3927c3927
< v5 = staticinvoke <org.apache.hadoop.hdfs.server.datanode.DataNode$lambda_incrDatanodeNetworkErrors_1__46: java.util.function.BiFunction bootstrap$()>();
---
> v5 = staticinvoke <org.apache.hadoop.hdfs.server.datanode.DataNode$lambda_incrDatanodeNetworkErrors_1__86: java.util.function.BiFunction bootstrap$()>();
